qi-doc/scribblings/field-guide.scrbl

```diff
@@ -8,18 +8,24 @@
                     racket]]
 
 @(define eval-for-docs
```

Comment:
There wasn't a specific reason, but when I tried to change it to use a single evaluator, it was initially getting some unexpected results because I think the evaluator was stateful across Scribble modules. I've now made it so that all of the modules use a common "factory" for creating the evaluator they use.